src/nginxconfig/templates/domain_sections/logging.vue

```diff
@@ -39,6 +39,32 @@ THE SOFTWARE.
                             </PrettyCheck>
                         </div>
                     </div>
+                    <div v-if="$props.data.accessLog.computed" class="control field is-horizontal is-expanded">
```

Comment:
(Not for this line) The parent `field is-horizontal` here likely needs a `is-aligned-top` added to the class list, and the `field-label` a `has-small-margin-top` to fix the alignment of the label with the checkbox still